di_website/whatwedo/blocks.py

```diff
@@ -81,3 +82,24 @@ class ExpertiseBlock(StructBlock):
 
     class Meta():
         template = 'blocks/expertise.html'
+
+
+class MapBlock(StructBlock):
+    """
+    Allows the addition of a single benefit
+    """
+    location = TextBlock()
+    address = TextBlock()
```

Comment:
Should this be required?

Reply:
`address = TextBlock()` should not be required, however `location = TextBlock()` is needed on the map to show the country name on mouse hover.

Reply:
Just replaced `addresss` with an optional `mobile` field` which can be also be shown on the map.